### DISCUSSION TITLE
한국 필터가 전부 사라졌습니다. / All of the South Korea filters are gone.

### DISCUSSION THREAD

**Discussion by namuhar** [List-KR/community#54] [Q&A]
![스크린샷 2024-02-02 024112](https://github.com/List-KR/community/assets/158504141/47db2952-40a9-4305-be31-9d88e19f8c59)
![스크린샷 2024-02-02 024146](https://github.com/List-KR/community/assets/158504141/6522d7b7-b963-4d86-a8d2-eb6ff5f9b0a0)
![스크린샷 2024-02-02 161827](https://github.com/List-KR/community/assets/158504141/d5af9296-31e8-4e64-aa18-7a25c2589854)

2024년 2월 2일 2시 기준

한국 필터가 전부 사라졌습니다. 다른 나라 필터들은 사용가능한데 한국만 404 not found으로 표시 됩니다.

며칠전에도 비슷한 문제가 있었는데 몇시간 지나서 다시 정상적으로 돌아와서 넘어갔는데

이번에 한번 더 동일한 오류가 생겨서 질문드립니다.

혹시 해결법 아시는 분들은 답변 해주시면 감사하겠습니다.

As of 2:00 PM on February 2, 2024

All filters for South Korea have disappeared. The other country filters are still available, but only Korea is showing as 404 not found.

I had a similar problem a few days ago, but after a few hours it was back to normal and I passed.

This time, I'm getting the same error again.

If anyone knows a solution, I would appreciate it if you could answer.

**Comment by piquark6046** [comment]
https://cdn.jsdelivr.net/gh/List-KR/List-KR@latest/filter-uBlockOrigin.txt
위 내용이 잘 표시되나요?

**Reply by namuhar** [reply]
아뇨 눈 표시를 눌렀을때 첨부한 이미지대로 404 not found 로 표시됩니다.

업데이트(시계표시)를 눌러도 계속 0 개중 0개 사용으로 나와요

**Reply by piquark6046** [reply]
Firefox 쓰세요? Chromium 계열 쓰세요?

**Reply by namuhar** [reply]
크롬으로 사용하고 있습니다.

**Reply by namuhar** [reply]
![스크린샷 2024-02-02 161827](https://github.com/List-KR/community/assets/158504141/416a43ad-6420-41a7-8580-0e4cd480a7f4)
![스크린샷 2024-02-02 163110](https://github.com/List-KR/community/assets/158504141/5f921e6f-a045-4966-8d1d-7dfb61c9cdf4)

url 링크를 타고 https://cdn.jsdelivr.net/gh/List-KR/List-KR@latest/filter-uBlockOrigin.txt 를 눌렀을때 표시되는 페이지도 올렸습니다.

하지만 ublock origin 에서 한국어 필터 '눈표시' 를 눌렀을때는 404 not found로 표시됩니다.

보시다시피 일본 필터는 정상적으로 사용되어서 한국 필터쪽이 문제가 있지 않나하는 생각이 있습니다.

**Reply by piquark6046** [reply]
uBlock Origin를 다시 설치해보세요.

**Reply by namuhar** [reply]
와... 설마 이걸로 해결될줄은... 답변 감사드립니다. 혹시 재설치해야만 정상작동되는 이유를 알려주실 수 있으신가요?

**Reply by piquark6046** [reply]
저도 모릅니다. 내부에서 꼬인 것 같아서 재설치해보시라고 말씀드린겁니다.
